index.js

```diff
@@ -88,7 +92,7 @@ export default class Toast extends Component {
 
     render() {
         let pos;
-        switch (this.props.position) {
+        switch ('center') {
```

Comment:
Hi @imMayankRaj, thanks for this pull request. Just looking at the code it seems this change is not required? If we state `'center` in the switch condition the toast will always be shown in the center regardless of user's preference. Should we revert back this bit?

Reply:
Just verified this on my end and can confirm the tap functionality works on iOS. We still have to revert the line above, but besides that all seems good to me. @crazycodeboy would you be able to look into merging this request?